services/jwt/index.js

```diff
@@ -0,0 +1,25 @@
+import jwt from 'jsonwebtoken';
+
+/**
+ * Creates a JSON Web Token (JWT) for a given user.
+ * 
+ * @param {string} id - The unique identifier of the user.
+ * @param {string} secret - The secret key used to sign the token.
+ * @returns {Promise<object>} A promise that resolves to the generated JWT token.
+ */
+
+export const createToken = async (id, secret) => {
```

Comment:
Accept secret as parameter is not a the best practice, let's use the ENV variable here for the secret